rest/admin-api/src/main/java/org/keycloak/admin/api/AdminRootV2.java

```diff
@@ -0,0 +1,28 @@
+package org.keycloak.admin.api;
+
+import jakarta.ws.rs.OPTIONS;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.core.Context;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.services.resources.admin.AdminCorsPreflightService;
+
+@jakarta.ws.rs.ext.Provider
+@Path("v2")
+public class AdminRootV2 {
```

Comment:
I wonder if the basepath for the new API should include `admin` and/or `api` to make it easier for users to protect routes in reverse proxies etc.

E.g.: `http://${hostname}/admin/api/...`

Reply:
Yes, it makes sense to me to emphasize more the 'API' stuff. 

So the new Admin API might be accessible at `/admin/api/v2`.

We can even provide the possibility that if the version is not specified, it uses the latest version of the API - as the old API uses only `/admin` root path.